engine/src/test/scala/cromwell/engine/workflow/lifecycle/execution/ejea/PerTestHelper.scala

```diff
@@ -137,6 +135,7 @@ private[ejea] class PerTestHelper(implicit val system: ActorSystem) extends Mock
       ioActor = ioActorProbe.ref,
       jobStoreActor = jobStoreProbe.ref,
       callCacheReadActor = callCacheReadActorProbe.ref,
+      callCacheWriteActor = callCacheWriteActorProbe.ref,
```

Comment:
This probe never seems to be inspected? Do we want to verify that it receives the appropriate messages (and the actors respond correctly to its messages)?